src/java/org/apache/cassandra/db/TableWriteHandler.java

```diff
@@ -23,5 +23,5 @@
 
 public interface TableWriteHandler
 {
-    void write(PartitionUpdate update, WriteContext context, UpdateTransaction updateTransaction);
+    void write(PartitionUpdate update, WriteContext context, boolean updateIndexes);
```

Comment:
I think this indirectly breaks index API compatibility, which is fine if this is going to ship in a major.